content/python/concepts/built-in-functions/terms/eval/eval.md

```diff
@@ -0,0 +1,44 @@
+---
+Title: 'eval()'
```

Comment:
Not sure if this standard has been changed but I believe we include the period: '.eval()'

Reply:
@caupolicandiaz: It's a function, not a method. No period.